Source/bmalloc/bmalloc/ProcessCheck.mm

```diff
@@ -37,13 +37,13 @@ bool gigacageEnabledForProcess()
     // Note that this function is only called once.
     // If we wanted to make it efficient to call more than once, we could memoize the result in a global boolean.
 
-    NSString *appName = [[NSBundle mainBundle] bundleIdentifier];
+    NSString *appName = [NSBundle mainBundle].bundleIdentifier;
```

Comment:
`NSBundle.mainBundle.bundleIdentifier`?

Reply:
NSBundle is the class name, not a variable name

Reply:
You can still do `NSBundle.mainBundle.bundleIdentifier`